### PR TITLE
valkyrie:0.2.0

### PR DESCRIPTION
**Pull request by JamesxX**
I am submitting
- [ ] a new package
- [x] an update for a package

Description: Valkyrie is a type validation package akin in typescript's Zod. In this 0.2.0 update, a plethora of additional types have been added, along with support for custom assertions and coercions. Groundwork has been laid to support community-defined schemas.

As part of 0.2.0, the project has been rehoused on the typst-community GitHub organisation.

### CODE REVIEW

**Comment by tingerrr** [comment]
I'd like to wait with the merge before the last comments I left on [valkyrie#5](https://github.com/typst-community/valkyrie/pull/5) are resolved, at least the manual monospace font issue.

**Comment by JamesxX** [comment]
> I'd like to wait with the merge before the last comments I left on [valkyrie#5](https://github.com/typst-community/valkyrie/pull/5) are resolved, at least the manual monospace font issue.

done

File: packages/preview/valkyrie/0.2.0/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# [unreleased](https://github.com/jamesxX/valakyrie/releases/tags/)
```
Comment (elegaanz) [inline packages/preview/valkyrie/0.2.0/CHANGELOG.md:1]: Shouldn't that say `0.2.0`?

Comment (elegaanz) [reply packages/preview/valkyrie/0.2.0/CHANGELOG.md:1]: Also, the links in this file still point to the old repository.

**Review by elegaanz — COMMENTED** [review]
Just a remark about the changelog, but I'm not even sure it is relevant as most people will read it from the typst-community repo and not from here.

File: packages/preview/valkyrie/0.2.0/CHANGELOG.md (hunk 1 above)
Comment (JamesxX) [reply packages/preview/valkyrie/0.2.0/CHANGELOG.md:1]: Resolved by removing changelog from this PR (you're right that this is unlikely to be the place they look).

**Comment by elegaanz** [comment]
Thanks :)

**Pull request merged**